### ISSUE TITLE
--handle-input not working with Expo?

### ISSUE THREAD

**Issue by FezVrasta** [open-cli-tools/concurrently#516]
- Description: I'm trying to run `expo start` along with another command, I'm using `--handle-input` but nothing happens when I press `r` (reload) or `i` (open on iOS)
- Expected Behavior: I would expect the commands to be forwarded to the Expo CLI
- Environment: macOS, Node lts/jod
- Reproduction: `concurrently -k -n 'mobile-client,backend' -c 'red,green' --handle-input 'yarn expo start' 'yarn workspace backend run dev'`

**Comment by gustavohenke** [comment]
Sounds like it's detecting whether it's in TTY mode. I don't know how this is used, but the function name is pretty suggestive:

https://github.com/expo/expo/blob/988261c1a1405aca1f0a32f28bf068306aa03a8c/packages/%40expo/cli/src/utils/interactive.ts#L4-L5

If you pass the `--raw` flag to concurrently, expo might respond - but at the cost of not having output prefixed.

It'd be cool to have some env variable to force interactivity.

**Comment by FezVrasta** [comment]
Thanks. Do you think the raw option could be enabled just for a specific command rather than for all of them? So that I can have the output prefixed for all the other commands I run?

**Comment by gustavohenke** [comment]
Yep, this should already be possible today though through API mode.

```js
concurrently([
	{ name: '...', command: 'foo', raw: true },
	{ name: '...', command: 'bar' } // raw: false by default
])
```

[Docs](https://github.com/open-cli-tools/concurrently#concurrentlycommands-options) (it's lacking `raw` in there, but it works)

That said, I wouldn't say no to making the CLI `--raw` flag accept indices/names as a way to pick which commands use it.

**Comment by FezVrasta** [comment]
so would it be `--raw 0` and/or `--raw label-a` right? Could it support multiple raws at the same time? (`--raw 0 --raw 1`)?

**Comment by gustavohenke** [comment]
Yes, but I'd favour using comma separators like in most other flags that accept multiple inputs (e.g. `--prefix-colors`, `--names` or `--success`), so `--raw foo,bar,1,2,3`

**Comment by ByScripts** [comment]
Any news on that issue?

I'm using `concurrently` to run multiple Vue apps at once (`concurrently "pnpm:dev:*"`) and I am unable to send command to any of them.

**Comment by gustavohenke** [comment]
Hey @ByScripts, did you try with `--raw`?
If it works, it'll confirm that your `dev:*` scripts are detecting it's not a TTY, and therefore ignore input forwarding.
